package.json

```
@@ -8,9 +8,11 @@
     "start": "astro dev",
     "build": "astro build",
     "preview": "astro preview",
-    "astro": "astro"
+    "astro": "astro",
+    "genI18N": "astro-i18next generate"
```

This is just a suggestion: It is common to name the scripts using kebab-case.

```suggestion
    "gen-i18n": "astro-i18next generate"
```